cmd/lima-guestagent/daemon_linux.go

```diff
@@ -21,6 +21,8 @@ func newDaemonCommand() *cobra.Command {
 		RunE:  daemonAction,
 	}
 	daemonCommand.Flags().Duration("tick", 3*time.Second, "tick for polling events")
+	daemonCommand.Flags().Bool("audit", true, "use audit features")
+	daemonCommand.Flags().Bool("tcp", false, "use tcp server instead a UNIX socket")
```

Comment:
I heard modern Windows supports Unix sockets

Reply:
Also wondering if we can use virtio-serial or vsock

Reply:
Windows does support Unix sockets (AF_UNIX), and I tried getting them to work, but I couldn't. I could create the socket on either side, but I couldn't get Windows to recognize a socket created by the guestagent (or the other way around). Would love to know if anyone else has experience or guidance here.

I think vscock could work though, I'm looking into it.

Reply:
Adde vsock support, turned out to be pretty interesting @AkihiroSuda